<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>xacnhanvanban</class>
 <widget class="QDialog" name="xacnhanvanban">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1057</width>
    <height>514</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Xác nhận văn bản</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Văn bản</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>cần xác nhận</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>280</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Chữ ký</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_chuKy">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>250</y>
     <width>651</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Khóa công khai</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_khoaCongKhai">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>350</y>
     <width>651</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_taiVanBanTXT">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>470</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Tải VB .txt</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_taiVanBanDocx">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>470</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Tải VB .docx</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_taiChuKy">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>470</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Tải chữ ký</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_xacNhan">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>470</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Xác nhận</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_thoat">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>470</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Thoát</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_vanBanCanXacNhan">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>651</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_taiKhoaCongKhai">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>470</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Tải khóa công khai</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_giaiMa">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>30</y>
     <width>211</width>
     <height>401</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
